flink-staging/flink-streaming/flink-streaming-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSource.java

```
@@ -102,4 +112,18 @@ public void run(SourceContext<OUT> ctx) throws Exception {
 	public void cancel() {
 		running = false;
 	}
+
+	@Override
+	protected void acknowledgeIDs(List<Long> ids) {
+		try {
+			channel.basicAck(ids.get(ids.size() - 1), true);
+		} catch (IOException e) {
+			e.printStackTrace();

```

Exceptions should never be swallowed.